lib/components/table.vue

```diff
@@ -76,14 +76,15 @@
 </template>
 
 <script>
-    import {warn} from '../utils';
+    import { warn } from '../utils';
+    import { keys } from '../utils/object';
```

Comment:
Es2015 imports are definitely not the same as object destructuring. I've been down that road. It stinks!